### PR TITLE
Update installation guide for cuTENSOR on Conda-Forge

### PR DESCRIPTION
**Pull request by leofang**
This can be backported.

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by leofang** [comment]
pfnCI, test this please

File: docs/source/install.rst [hunk 1]
```diff
@@ -130,7 +130,12 @@ the installation of a particular CUDA version (say 10.0) for driver compatibilit
 
 .. note::
 
-    Currently cuTENSOR is not yet available on ``conda-forge``.
+    cuTENSOR is available on ``conda-forge`` for CUDA 10.1+ and is an optional dependency. To install CuPy with the cuTENSOR
+    support enabled, you can do::
+
+    $ conda install -c conda-forge cupy cutensor cudatoolkit=10.2
+
+    ``cupy`` and ``cutensor`` must coexist in order for the ``conda`` solver to pick the right package.
```
Comment (kmaehashi) [inline docs/source/install.rst:138]: Does "coexist" mean "needs to be installed in a single command"?
Sounds a bit ambiguous to me.

Comment (leofang) [reply docs/source/install.rst:138]: Indeed. How about "CuPy and cutensor must be installed at the same time"?

Comment (kmaehashi) [reply docs/source/install.rst:138]: Sounds good to me!

Comment (leofang) [reply docs/source/install.rst]: Thanks! Done in 866ab84.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Pull request merged**